test/pylib/util.py

```diff
@@ -29,19 +31,18 @@ def process(self, msg, kwargs):
         return '[%s] %s' % (self.extra['prefix'], msg), kwargs
 
 
-unique_name_prefix = 'test_'
 T = TypeVar('T')
 
 
-def unique_name():
+def unique_name(unique_name_prefix = 'test_'):
     if not hasattr(unique_name, "last_ms"):
         unique_name.last_ms = 0
     current_ms = int(round(time.time() * 1000))
     # If unique_name() is called twice in the same millisecond...
     if unique_name.last_ms >= current_ms:
         current_ms = unique_name.last_ms + 1
     unique_name.last_ms = current_ms
-    return unique_name_prefix + str(current_ms)
+    return unique_name_prefix + str(current_ms) + '_' + ''.join(random.choice(string.ascii_lowercase) for _ in range(5))
```

Comment:
>  test: extend unique_name with random infix 

This is no longer a random infix but rather a suffix, right? Please update the commit message.

Reply:
yes